src/vcpkg/commands.x-forward.cpp

```diff
@@ -0,0 +1,21 @@
+#include <vcpkg/base/system.process.h>
+
+#include <vcpkg/commands.x-forward.h>
+#include <vcpkg/vcpkgcmdarguments.h>
+
+namespace vcpkg::Commands::X_Forward
+{
+    void perform_and_exit(const VcpkgCmdArguments& args, const VcpkgPaths&)
+    {
+        Command cmd("vcpkg-utility-impl");
```

Comment:
where does `vcpkg-utility-impl` come from?

Reply:
It comes from nowhere -- it's a demonstrative "insert executable name here".